docs/tutorials/bucket_tutorial.jl

```diff
@@ -0,0 +1,315 @@
+# at the surface of the earth.
+
+# Here we define the model drivers, starting with downward radiation.
+SW_d =
+    (t) -> eltype(t)(
+        sin(2.0 * π * t / 86400) > 0 ? 300.0 * sin(2.0 * π * t / 86400) : 0.0,
+    );
+LW_d =
+    (t) -> eltype(t)(
+        sin(2.0 * π * t / 86400) > 0 ? 300.0 * sin(2.0 * π * t / 86400) : 0.0,
+    );
+bucket_rad = PrescribedRadiativeFluxes(FT, SW_d, LW_d);
+
+# Prescribed atmospheric variables
+
+# Stochastic prescipitation:
+precip = (t) -> eltype(t)(5e-7 * rand() * (rand() > 0.97));
+# Diurnal temperature variations:
+T_atmos = (t) -> eltype(t)(300.0 + 5.0 * sin(2.0 * π * t / 86400 + 7200));
+# Constant otherwise:
+u_atmos = (t) -> eltype(t)(3.0);
+q_atmos = (t) -> eltype(t)(0.005);
+h_atmos = FT(10);
+ρ_atmos = (t) -> eltype(t)(1.13);
+ρ_sfc = FT(1.15);
+bucket_atmos = PrescribedAtmosphere(
+    precip,
+    T_atmos,
+    u_atmos,
+    q_atmos,
+    ρ_atmos,
+    h_atmos,
+    ρ_sfc,
+);
+
+# Then, we create the model object, which contains the drivers, parameters,
+# domain, and is associated with the correct differential equations
+# for the bucket model:
+model = BucketModel(
+    parameters = bucket_parameters,
+    domain = bucket_domain,
+    atmosphere = bucket_atmos,
+    radiation = bucket_rad,
+);
+
+# Note the holder structs for the radiation and atmosphere functions: they
+# are named `Prescribed`. In coupled simulations, we would use a different
+# type and rely on multiple dispatch to obtain the atmospheric and radiative
+# quantitites from the coupler.
+
+# Like all ClimaLSM models, we set up the state vector using `initialize`:
+Y, p, coords = initialize(model);
+
+# We can inspect the prognostic and auxiliary variables of the model:
+ClimaLSM.prognostic_vars(model)
+Y.bucket |> propertynames
+# The auxiliary variables in this case are the turbulent fluxes, the
+# net radiation, and the surface specific humidity.
+ClimaLSM.auxiliary_vars(model)
+p.bucket |> propertynames
+
+
+# Next is to set initial conditions. 
+Y.bucket.T_sfc .= FT(286.5);
+Y.bucket.W .= FT(0.1);
+Y.bucket.Ws .= FT(0.0);
+Y.bucket.S .= FT(0.0);
+
+# Then to create the entire right hand side function for the system
+# of ordinary differential equations:
+ode_function! = make_ode_function(model);
+
+# Then set up the simulation
+t0 = FT(0.0);
+tf = FT(3 * 86400);
+prob = ODEProblem(ode_function!, Y, (t0, tf), p);
+Δt = FT(1000.0);
+# We need a callback to get and store the auxiliary fields, as they
+# are not stored by default.
+saved_values = SavedValues(FT, ClimaCore.Fields.FieldVector);
+cb = SavingCallback(
+    (u, t, integrator) -> copy(integrator.p),
+    saved_values;
+    saveat = 0:Δt:tf,
+);
+
+sol = solve(prob, Euler(); dt = Δt, saveat = 0:Δt:tf, callback = cb);
+
+# Extractin the solution from what is returned by the ODE.jl commands
+# is a bit clunky right now, but we are working on hiding some of this.
+# `parent` extracts the underlying data from the ClimaCore.Field.FieldVector,
```

Comment:
ClimaCore.Field*s*. Also, would `parent()` of a `FieldVector` work? I've only ever applied it to individual `Field` objects.

Reply:
I dont think that it would! but ive never tried either. I changed the language to be clearer.